### PR TITLE
Improve #6125: Place path in park entrances

### PR DESCRIPTION
**Pull request by trigger-death**
Can now update park entrance path by placing path over the middle tile.
Updating the path costs money just like normally replacing a path

Updating the path type can only be done in sandbox mode or in the
scenario editor.

Added feature notice to changelog.

Preview

### CODE REVIEW

File: src/openrct2/world/footpath.c [hunk 1]
```diff
@@ -513,7 +543,8 @@ void game_command_place_footpath(sint32 *eax, sint32 *ebx, sint32 *ecx, sint32 *
 
 static money32 footpath_place_from_track(sint32 type, sint32 x, sint32 y, sint32 z, sint32 slope, sint32 edges, sint32 flags)
```
Comment (Gymnasiast) [inline src/openrct2/world/footpath.c]: Should a track design really be able to replace the entrance path?

File: src/openrct2/world/footpath.c [hunk 2]
```diff
@@ -173,8 +173,9 @@ static const uint8 byte_98D7EC[] = {
 
 static money32 footpath_element_insert(sint32 type, sint32 x, sint32 y, sint32 z, sint32 slope, sint32 flags, uint8 pathItemType)
 {
-    rct_map_element *mapElement;
+    rct_map_element *mapElement, *entranceElement;
```
Comment (Gymnasiast) [inline src/openrct2/world/footpath.c]: Needs a space after the `*`.

File: src/openrct2/world/footpath.c (hunk 1 above)
Comment (trigger-death) [reply src/openrct2/world/footpath.c]: There's no reason why you shouldn't be able too. It's the same type of change for placing path normally.

**Comment by Gymnasiast** [comment]
Could you rebase this and make sure the network version is bumped after that?

**Pull request merged**